tianshou/env/worker/subproc.py

```diff
@@ -104,7 +104,7 @@ def _encode_obs(
             elif cmd == "render":
                 p.send(env.render(**data) if hasattr(env, "render") else None)
             elif cmd == "seed":
-                p.send(env.seed(data) if hasattr(env, "seed") else None)
+                env.reset(seed=data)
```

Comment:
Why not support arbitrary `*args, **kwargs` at L90?

Reply:
good suggestion, will support that as well.